Vault-Deployment/values/vault-values.yaml

```diff
@@ -46,53 +44,37 @@ server:
     name: hashicorp-vault-license
     readOnly: true
 
-
-
-
-  standalone:
```

Comment:
Let's keep it simple for the demo purposes here. Since we have 1 node only, let's just go with the affinity rules in the default config @Xelef2000